test/unit/cog_api/fake/users_test.exs

```diff
@@ -48,6 +48,25 @@ defmodule CogApi.Fake.UsersTest do
       assert Enum.map(found_user.groups, &(&1.id)) == [group.id]
       assert List.first(found_user.groups).roles == [role]
     end
+
+    it "searches by username" do
```

Comment:
The other file was using a `context` already for something else. I looked again, and it made sense to throw it inside _that_ context, so I "fixed" it there.